pyanaconda/modules/common/base/base.py

```diff
@@ -104,6 +128,21 @@ def publish_task(self, namespace, task, message_bus=DBus):
         object_path = publish_task(message_bus, namespace, task)
         self.published_tasks[task] = object_path
 
+
+class KickstartModule(MainModule, KickstartBaseModule):
+    """Implementation of a main kickstart module.
+
+    The main kickstart module is able to porse and generate the given
+    kickstart string based on its kickstart specification.
+    """
+
+    def __init__(self):
+        MainModule.__init__(self)
+        KickstartBaseModule.__init__(self)
```

Comment:
I'm just wondering why not to use super here as well but guess you have a good reason.

Reply:
`super().__init__()` will call only `MainModule.__init__(self)`

Reply:
Fixed.